### PR TITLE
Add e2e testing workshop

### PR DESCRIPTION
**Pull request by jerivas**

### CODE REVIEW

**Comment by jerivas** [comment]
[Link Ed's Python Workshop](https://trello.com/c/BtMHm9Ql/199-link-eds-python-workshop)

File: content/talks/django-end-to-end-testing.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - Django
+  - Python
+slides: https://drive.google.com/file/d/1K3Fyn4uj2uZoJSO5H5mFCMXr6206kiNa/views
+events:
+  - venue: DjangoCon US 2022
+    adr: San Diego, CA (hybrid)
+    url: https://2022.djangocon.us/tutorials/effective-end-to-end-testing-for-django/
+    date: 2022-10-16
+    end: 2022-10-21
+    slug: 2022-djangocon
```
Comment (jerivas) [inline content/talks/django-end-to-end-testing.md:23]: I'm not sure what this `slug` field does. Let me know if it's wrong 😅

Comment (jgerigmeyer) [reply content/talks/django-end-to-end-testing.md:23]: I think this is only used for press quotes, not events.

```suggestion
```

File: content/talks/django-end-to-end-testing.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Effective end-to-end testing for Django
```
Comment (jgerigmeyer) [inline content/talks/django-end-to-end-testing.md:2]:
```suggestion
title: Effective End-to-End Testing for Django
```

File: content/talks/django-end-to-end-testing.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+following topics with practical exercises that each participant completes on
+their own computer:
+
+- "User contract" testing
+- Pytest and custom fixtures
+- Asserting DOM state
+- Testing interactive elements
+- Integrating your test suite with Continuous Integration (CI)
+
+## Comments from attendees
```
Comment (jgerigmeyer) [inline content/talks/django-end-to-end-testing.md:49]:
```suggestion
## Comments From Attendees
```

File: content/talks/django-end-to-end-testing.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+> I liked that it was hands-on, started with basics, and had failing tests as
+> part of the tutorial. Seeing progression from failure to passing is really
+> helpful for understanding what is happening in the code.
+
+> Though I am not new to testing these techniques are new and super helpful. I’m
```
Comment (jgerigmeyer) [inline content/talks/django-end-to-end-testing.md:55]:
```suggestion
> Though I am not new to testing, these techniques are new and super helpful. I’m
```

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — CHANGES_REQUESTED** [review]
This is looking great!

- I heard your tutorial referred to as a "workshop" by various people. If it seems more like a workshop than a talk, you could switch the tag from "talk" to "workshop."
- I don't know why, but the slides link at the top right of the page doesn't work for me. (The one in the event section does work correctly.) I get an error from Google Drive saying, "Sorry, unable to open the file at this time."

File: content/talks/django-end-to-end-testing.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  alt: |
+    People gathered around a table with multiple computers, cellphones, and
+    notebooks on top.
+  position: top
+tags:
+  - Testing
+  - Tutorial
+  - Django
+  - Python
+slides: https://drive.google.com/file/d/1K3Fyn4uj2uZoJSO5H5mFCMXr6206kiNa/views
```
Comment (jgerigmeyer) [inline content/talks/django-end-to-end-testing.md:16]:
```suggestion
slides: https://drive.google.com/file/d/1K3Fyn4uj2uZoJSO5H5mFCMXr6206kiNa/view
```

File: content/talks/django-end-to-end-testing.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  This tutorial provides hands-on experience with end-to-end testing for Django
+  using pytest and Playwright. Participants acquire solid strategies to test
+  common patterns in Django applications and the experience of writing a
+  complete test suite for a non-trivial application.
+---
+
+A comprehensive test suite gives you the confidence to fix bugs, refactor, and
+add new features to your application without fear of unknowingly breaking
+unrelated parts of the codebase. End-to-end tests are a useful and important
+member of your testing toolbox and tools like [pytest](https://pytest.org) and
```
Comment (jgerigmeyer) [inline content/talks/django-end-to-end-testing.md:34]:
```suggestion
member of your testing toolbox, and tools like [pytest](https://pytest.org) and
```

**Comment by jgerigmeyer** [comment]
> * I heard your tutorial referred to as a "workshop" by various people. If it seems more like a workshop than a talk, you could switch the tag from "talk" to "workshop."

(I think that would probably mean moving from the `talks` to the `workshops` folder.)

**Review by jgerigmeyer — APPROVED** [review]
:shipit:

**Comment by jgerigmeyer** [comment]
@SondraE Could you re-review this PR now that Ed addressed your comments?

**Review by SondraE — APPROVED** [review]
lgtm!

**Pull request merged**